packages/next-mdx/readme.md

```diff
@@ -1,78 +0,0 @@
-# Next.js + MDX
```

Comment:
We should keep the file here but link to the new plugin. Also link to the old docs (a previous commit) if the user wants to read about the old usage.

Reply:
Updated

Reply:
👌